python/pyspark/accumulators.py

```diff
@@ -262,9 +262,10 @@ def authenticate_and_accum_updates():
                 raise Exception(
                     "The value of the provided token to the AccumulatorServer is not correct.")
 
-        # first we keep polling till we've received the authentication token
-        poll(authenticate_and_accum_updates)
-        # now we've authenticated, don't need to check for the token anymore
+        if auth_token:
```

Comment:
nit: I _think_ it's better better style (and consistent with the rest of the changes) to use `if autho_token is not None`.

Reply:
+1